README.md

```diff
@@ -63,6 +63,14 @@ This is often used for node-recycling scenarios, as seen in [the example below](
 
 _We are discussing the naming and API for this functionality in [#25](https://github.com/valdrinkoshi/virtual-list/issues/25)._
 
+### `itemKey` property
+
+Type: `function(child: Element) => string|number`

```

Comment:
string|number is strange. We should state how this is actually sorted. What if I mix strings and numbers? Does this just use JS's `array.sort()`?

Reply:
`itemKey` doesn't affect the sorting directly, as it just provides an identifier for a given item. `Repeats` uses this identifier to figure out if we should generate the DOM or not (if to invoke `newChild`)

Reply:
Oh, right, that makes sense.

So why does it have to be a string or number? Couldn't it be anything?

Reply:
...indeed! Will update the signature here :)

Reply:
What's an efficient way to say that it should be defined in the signature?
` => any` makes it look like it can be null or undefined

Reply:
I think `=> any` is correct. If you return `null` or `undefined` that's bad in the same way that always returning `1` is bad.

Eventually we'll need to formally specify what happens if you do bad things, but for now we can just document the happy path.

Reply:
sg. Also, I realized that the input was incorrect, the function doesn't receive `Element` but the data itself...Corrected!